src/jit/compiler.hpp

```diff
@@ -150,9 +150,9 @@ unsigned __int64 genFindHighestBit(unsigned __int64 mask)
 #endif // 0
 
 /*****************************************************************************
-*
-*  Return true if the given 64-bit value has exactly zero or one bits set.
-*/
+ *
```

Comment:
What's up with all these changes?

Reply:
jit-format doing its magic -- not sure why.

Reply:
jit-format is also doing thsi for me too